src/Examples/UnitModels/Operations/translator_testing.ipynb

```diff
@@ -0,0 +1,355 @@
+    "\n",
+    "Benzene Molar Concentration = mol/m^3\n",
+    "\n",
+    "Toluene Molar Concentration = mol/m^3\n",
+    "\n",
+    "Temperature = 298 K\n",
+    "\n",
+    "Pressure = 101325 Pa\n",
+    "\n",
+    "\n",
+    "For more details, please refer to the IDAES documentation: https://idaes-pse.readthedocs.io/en/stable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up the problem in IDAES"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following cell, we will be importing the necessary components from Pyomo and IDAES."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import objects from pyomo package \n",
+    "from pyomo.environ import ConcreteModel, Constraint, value, SolverFactory\n",
+    "\n",
+    "# Import the solver\n",
+    "from idaes.core.solvers import get_solver\n",
+    "\n",
+    "# Import the main FlowsheetBlock from IDAES. The flowsheet block will contain the unit model\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "\n",
+    "# Import the product unit model\n",
+    "from idaes.models.unit_models import Translator\n",
+    "\n",
+    "# Import idaes logger to set output levels\n",
+    "import idaes.logger as idaeslog\n",
+    "\n",
+    "# Import the modular property package to create a property block for the flowsheet\n",
+    "from idaes.models.properties.modular_properties.base.generic_property import GenericParameterBlock\n",
+    "\n",
+    "# Import the BT_ideal property package to create a configuration file for the GenericParameterBlock based on the ideal eos\n",
+    "from idaes.models.properties.modular_properties.examples.BT_ideal import configuration as configuration_ideal\n",
+    "\n",
+    "# Import the BT_PR property package to create a configuration file for the GenericParameterBlock based on the Peng-Robinson eos\n",
+    "from idaes.models.properties.modular_properties.examples.BT_PR import configuration as configuration_PR\n",
+    "\n",
+    "# Import the degrees_of_freedom function from the idaes.core.util.model_statistics package\n",
+    "# DOF = Number of Model Variables - Number of Model Constraints\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "\n",
+    "# Create the ConcreteModel and the FlowsheetBlock objects, and attach the flowsheet block to it.\n",
+    "m = ConcreteModel()\n",
+    "\n",
+    "m.fs = FlowsheetBlock(dynamic=False) # dynamic or ss flowsheet needs to be specified here\n",
+    "\n",
+    "# Add properties parameter block to the flowsheet with the BT_ideal configuration file\n",
+    "m.fs.properties_ideal = GenericParameterBlock(**configuration_ideal)\n",
+    "\n",
+    "# Add properties parameter block to the flowsheet with the BT_PR configuration file\n",
+    "m.fs.properties_PR = GenericParameterBlock(**configuration_PR)"
```

Comment:
Still a bit unsure which two property packages I should translate to and from. It seems like all of the modular property packages use the same state variables (FTPx). So in this example, I think the translator block is functioning correctly, it's just not really doing anything since all of the state variables are the same.

Reply:
Ah, you need to get an example which uses a different set of state variables. If you look in the tests for the modular properties, you can find some examples using different states - you will need to create a separate configuration dictionary somewhere however (either in the notebook, or as a separate file nearby).

Reply:
Not sure which would be better - putting the configuration dictionary in the notebook or a separate file nearby. As of right now, they are in the notebook. Some of the pros of this are that everything is in one place and showing how the configuration dictionary is set up here could be convenient for a user that wants to create their own. The major con I see is that it may make the translation unit model file seem overly complex.